### ISSUE TITLE
Details differ between `symfony` preset and PHP Coding Standards Fixer

### ISSUE THREAD

**Issue by dbohdan** [lkrms/pretty-php#131]
I have compared `--preset symfony` with PHP CS Fixer and found two minor differences. I thought you may be interested.

## Original

```php
<?php declare(strict_types=1);

final class Foo {
    public static function bar() { }
}
```

## pretty-php v0.4.66-d7e66927

```php
<?php declare(strict_types=1);

final class Foo
{
    public static function bar() {}
}
```

## PHP CS Fixer 3.62.0 (1ce9e24)

```php
<?php

declare(strict_types=1);

final class Foo
{
    public static function bar()
    {
    }
}
```

**Comment by lkrms** [comment]
Thanks for sharing 😊

`pretty-php` presets are supposed to minimise the size of the commit that would be required to re-format the source code of the project they're named after, i.e. they're a bit experimental at the moment.

The Symfony codebase doesn't use `<?php declare(strict_types=1);`, which is why it currently remains collapsed as per `pretty-php`'s default behaviour, but if you add `--psr12` this becomes consistent with PHP CS Fixer's default settings.

Collapsing empty braces to the same line as the declaration they belong to is a bit more contentious, but I believe it's PER-compliant. Do you know if PHP CS Fixer un-collapses them by default, or does it leave them as-is? `pretty-php` collapses them by default, but if PHP CS Fixer does the opposite, it would make sense to adjust the Symfony preset, at least.

**Comment by dbohdan** [comment]
> but if you add `--psr12` this becomes consistent with PHP CS Fixer's default settings.

I see. Thanks for explaining.

> Do you know if PHP CS Fixer un-collapses them by default, or does it leave them as-is?

It does uncollapse them by default. While I prefer pretty-php's `{}` for the look and readability, it is probably best to adjust the Symfony preset.

**Comment by lkrms** [comment]
Both issues are fixed (no `--psr12` necessary) in v0.4.67.

**Comment by dbohdan** [comment]
Yay, thanks! It works for me. I can close this issue now.
